pyproject.toml

```
@@ -78,6 +78,7 @@ test = [
 packages = [
   "dandiapi",
 ]
+force-include = { "requirements.txt" = "requirements.txt" }
```

Does `include` work: https://hatch.pypa.io/1.9/config/build/#patterns

Or is that incompatible with the use of `packages`?
```suggestion
include = [ "requirements.txt" ]
```

---

I see the issue:
> `packages` option is semantically equivalent to `only-include`

> You can use the `only-include` option to prevent directory traversal starting at the project root and only select specific relative paths to directories or files. Using this option ignores any defined `include patterns`.

---

We should be able to do this instead:
```suggestion

# When building an sdist for upload to Heroku, also include "requirements.txt"
[tool.hatch.build.targets.sdist]
only-include = [
  "dandiapi",
  "/requirements.txt",
]
```

Technically, we could put this into `tool.hatch.build` (so that it applies when building any kind of install), but this makes it obvious that this is an explicit workaround for sdists being sent to Heroku.

---

Thanks, can confirm that `requirements.txt` is properly bundled into the sdist with this change :+1: